doc/developer.texinfo

```diff
@@ -2502,15 +2615,28 @@ If the web search is not successful, email the maintainer of the old content.
 @end itemize
 
 Note that you must fix it in the original repository,
-not in the drop directories.
-This means correcting the file, most likely located beneath either
-@file{$DAIKONDIR/doc} or @file{$BIBDIR}, committing the changes, and
-returning to the previous step (@command{make staging}) prior to re-running
-the broken link tool.
+not in the drop directories. This means:
+@itemize
+@item
+Switch back to your original Bash shell.
+@item
+Correct the file. (Most likely located beneath either
+@file{$DAIKONDIR/doc} or @file{$BIBDIR}.)
+@item
+Commit the changes.
+@item
```

Comment:
I think it's simpler to do the whole "make staging" process (from creating the new command shell) all over again.  It might take a bit longer in wall clock time, but it's no more human time and likely less.